cmd/internal/cli/apptainer.go

```diff
@@ -845,7 +845,7 @@ func getLibraryClientConfig(uri string) (*libClient.Config, error) {
 		if endpoint.DefaultLibraryURI != "" {
 			sylog.Warningf("no default remote in use, falling back to default library: %s", endpoint.DefaultLibraryURI)
 		} else {
-			return nil, fmt.Errorf("no default remote with library client in use")
+			return nil, fmt.Errorf("no default remote with library client in use (\"https://apptainer.org/docs/user/latest/endpoint.html#restoring-pre-apptainer-library-behavior\")")
```

Comment:
I decided to make a shorter link, so use https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote instead.  Also add the word "see" before the link.